pennylane/templates/tensornetworks/mps.py

```diff
@@ -62,27 +92,27 @@ class MPS(Operation):
             import pennylane as qml
             import numpy as np
 
-            def block(block_weights, block_wires):
-                qml.CNOT(wires=[block_wires[0],block_wires[1]])
-                qml.Rot(block_weights[0],block_weights[1],block_weights[2],wires=block_wires[0])
-                qml.Rot(block_weights[3],block_weights[4],block_weights[5],wires=block_wires[1])
+            def block(weights, wires):
+                qml.CNOT(wires=[wires[0],wires[1]])
+                qml.RY(weights[0], wires=wires[0])
+                qml.RY(weights[1], wires=wires[1])
 
             n_wires = 4
-            loc = 2
-            n_params_block = 6
-            template_weights = [[1,2,3,4,5,6],[3,4,5,6,7,8],[4,5,6,7,8,9]]
+            n_block_wires = 2
+            n_params_block = 2
+            template_weights = [[0.1,-0.3],[0.4,0.2],[-0.15,0.5]]
 
-            dev= qml.device('default.qubit',wires=n_wires)
+            dev= qml.device('default.qubit',wires=range(n_wires))
             @qml.qnode(dev)
-            def circuit(weights):
-                qml.MPS(wires = range(n_wires),loc=loc,block=block, n_params_block=n_params_block, weights=weights)
+            def circuit(template_weights):
+                qml.MPS(range(n_wires),n_block_wires,block, n_params_block, template_weights)
```

Comment:
Should we include a call to `n_blocks` to indicate to the user how they can use it to get the right shape of the template weights?

Reply:
Added in commit https://github.com/PennyLaneAI/pennylane/pull/1871/commits/d6c7dc41bbaff2f5560af3947571640978a7f35c